requirements/dev.txt

```diff
@@ -23,3 +23,4 @@ tox>=3.21.0, <4.0.0 # used to generate licence info via `make licenses`
 twine>3.2.0, <4.0.0  # 4.x causes deps clashes with testing/requirements.txt, as requires rich>=12.0.0 -> requires typing-extensions>=4.0.0 -> too high for spacy and thinc!
 packaging>=19.0, <22.0 # Used to check scipy version for CVMDrift test. Can be removed once python 3.6 support dropped (and scipy lower bound >=1.7.0).
 codecov>=2.0.15, <3.0.0
+xgboost>=1.3.2, <2.0.0 # Install for use in testing since we support serialization of xgboost models under the sklearn API
```

Comment:
Need to install this for the xgboost model defined [here](https://github.com/SeldonIO/alibi-detect/pull/642/files#diff-18d404d6b5ac8c3a8692660e2de7eb57a989b7b260f7f15aecf0ed8ce09b4fe9R155) and used in a test [here](https://github.com/SeldonIO/alibi-detect/pull/642/files#diff-349b14885d352b4861af40d39a1a767f3794394c030d01173832d7277ec36900R14).